backend/src/controllers/userController.js

```diff
@@ -1,27 +1,23 @@
 import users from "../database/usersDb.js";
-import { fileURLToPath } from "url";
-import path from "path";
-import { hashSync, compareSync } from "bcrypt";
+import { hashSync } from "bcrypt";
 import { v4 as uuidv4 } from "uuid";
 const uniqueId = uuidv4();
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
-const filePath = path.join(
-  __dirname,
-  "../../../frontend/views/users-page.html"
-);
-
 const getUser = (req, res) => {
-  res.sendFile(filePath);
+  res.json(users);
```

Comment:
Should this be for getting users? Cus you are returning all users?

Reply:
Also returning all user information is not a good practice because hashed password should not be sent to the client. Even if it is hashed, it can be decrypted. For the safety of the users, password even when hashed should only be stored in the database.